packages/jest-config/index.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   testEnvironment: "node",
   testMatch: ["<rootDir>/src/**/*.test.ts"],
   transform: {
-    "^.+\\.ts?$": "esbuild-jest",
+    "^.+\\.tsx?$": "esbuild-jest",
```

Comment:
it looks like it will now only match .tsx files but not .ts, but we may have .ts too, no?

Reply:
It matches both. The `?` means the `x` can be present or not